src/test/cli/ceph-authtool/add-key.t

```diff
@@ -2,7 +2,7 @@
   creating kring
 
   $ ceph-authtool kring --add-key 'AQAK7yxNeF+nHBAA0SgSdbs8IkJrxroDeJ6SwQ== 18446744073709551615'
-  added entity client.admin auth auth(auid = 18446744073709551615 key=AQAK7yxNeF+nHBAA0SgSdbs8IkJrxroDeJ6SwQ== with 0 caps)
+  added entity client.admin auth(auid = 18446744073709551615 key = AQAK7yxNeF+nHBAA0SgSdbs8IkJrxroDeJ6SwQ==)
```

Comment:
maybe add a comma to better separate `auid` from `key`? i.e.

`added entity client.admin auth(auid = 18446744073709551615, key = AQAK7yxNeF+nHBAA0SgSdbs8IkJrxroDeJ6SwQ==)`

?